ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -161,6 +161,21 @@ literals:
 - supply_air_temperature_setpoint
 - supply_fan_current_sensor
 - supply_fan_power_sensor
+- supply_air_damper_open_status:
```

Comment:
is it possible to model this as one field `supply_air_damper_status` ? or are there legitimately two separate fields for open and closed?

Reply:
There are two fields. The status in Open field will be active only when the damper is fully open and the status in the Closed field will be active only when the damper is fully closed. If the damper is for example is stuck in an intermediate position, neither of the fields will be active.